pkgs/applications/editors/neovim/tests/default.nix

```diff
@@ -222,4 +223,24 @@ rec {
     export HOME=$TMPDIR
     ${nvimWithLuaPackages}/bin/nvim -i NONE --noplugin -es
   '';
+
+  # nixpkgs should install optional packages in the opt folder
+  nvim_with_opt_plug = neovim.override {
+    extraName = "-with-opt-plug";
+    configure.packages.plugins = with pkgs.vimPlugins; {
+      opt = [
+        (base16-vim.overrideAttrs(old: { pname = old.pname + "-unique-for-tests-please-dont-use-opt"; }))
+      ];
+    };
+    configure.customRC = ''
+      packadd base16-vim-unique-for-tests-please-dont-use-opt
```

Comment:
what would be nice is to check first that it fails without the packadd, see https://stackoverflow.com/questions/5698284/in-my-vimrc-how-can-i-check-for-the-existence-of-a-color-scheme/5702498#5702498 on how to do this.
Why the `set background=dark` ? seems superfluous here